packages/utils/src/typescript-installer.ts

```diff
@@ -14,22 +14,32 @@ export async function installAllTypeScriptVersions() {
   for (const v of TypeScriptVersion.shipped) {
     await install(v);
   }
+  if (TypeScriptVersion.shipped.length + 2 === TypeScriptVersion.supported.length) {
+    await install("rc");
+  }
   await installTypeScriptNext();
 }
 
 export async function installTypeScriptNext() {
   await install("next");
 }
 
-async function install(version: TsVersion | "next"): Promise<void> {
+async function install(version: TsVersion | "next" | "rc"): Promise<void> {
```

Comment:
One problem I see with installing `rc` without checking is that for 5.1, we added it to the array _before_ the rc was available, so if this runs, it'll get the previous rc from an old version and potentially make people think their code is working. Is that accurate?

Reply:
The check on line 17 means that it only happens during the RC period. I'll put a better comment.